src/SYS.ASM

```diff
@@ -255,29 +240,23 @@ command:                                        ; Main loop: Retrieve command fr
     cmp al, 0x3E                                ; Is this a forward-facing angle-bracket?
     je .redir                                   ; If so, set up output redirection
 
-.write:
-    stosb                                       ; Otherwise, store byte in list
-
-    mov bh, 0xFF                                ; We're currently reading a parameter
-    push di                                     ; Push new destination index
-    jmp .param_char                             ; And repeat
+    cmp al, 0x7C                                ; Is this a vertical bar?
+    je .pipe                                    ; If so, set up pipe
 
-.quotation:
-    stosb                                       ; Write the double-quote
+.parameter:
```

Comment:
By default, the kernel will now grab a raw parameter (with any quotations) and store it in a list, this eliminating the need to keep track of the destination pointer or whether a certain parameter is being written to.